lib/ecstatic.js

```diff
@@ -204,7 +213,7 @@ var ecstatic = module.exports = function (dir, options) {
         }
       }
 
-      if (path.extname(file) === '.gz') {
+      if (file === gzipped) {
```

Comment:
I know the current codebase isn't a shining example for this kind of thing, but how would you feel about making this a flag instead? `gzipped` being the filename and `isGzipped` being a boolean? I *think* that would be more clear.